pyClient/commands/zeth_deploy.py

```diff
@@ -43,14 +42,12 @@ def deploy(
     token_instance_desc = get_erc20_instance_description(token_address) \
         if token_address else None
 
-    zeth_client = MixerClient.deploy(
+    _zeth_client, mixer_instance_desc = MixerClient.deploy(
```

Comment:
What is  the leading  `_` in `_name_of_var` supposed to represent? (put otherwise, why did you rename `zeth_client` to `_zeth_client`?)

Reply:
For variables, the leading `_` marks the variable as unused preventing the linting tools from complaining.

Reply:
Oh alright I see. Isn't it possible to replace these unused variables by a `_` then? (i.e. `_, mixer_instance_desc = MixerClient.deploy(`

Reply:
AFAIK `_` is not treated as special, as it is in some languages, (except where it is used for "last value" in the REPL, IIRC) so it get's a value and type like any other variable. Hence it usually creates type conflicts if used multiple times per scope.  So I think it's possible, but not necessarily everywhere, and in Python it doesn't seem to have an advantage over the `_` prefixed version (which also includes information about the unused variable).  At least, that's my understanding.

Reply:
I see - it's not like in Go...
I just read that a way to get around that is to select the outputs of a function by doing something like `desired_output1, desired_output2 = function(args..)[indices_of_desired_outputs]`. Haven't tried it though...

Reply:
Interesting. I hadn't seen that approach before.